Cycling Analysis/Strava importer.ipynb

```diff
@@ -1270,28 +1359,48 @@
     "\n",
     "\n",
     "# Process each FIT file\n",
```

Comment:
Validating file paths before processing to avoid error and confusion.

Reply:
Once again, this code is run interactively by backend developers and the paths are set as variables earlier in the code. If the files don't exist then the code already shows output that the developer can understand.